tests/testsuite/install.rs

```diff
@@ -593,6 +593,48 @@ fn relative_install_location_with_trailing_slash() {
     assert_has_not_installed_exe(&project_t1, "foo");
 }
 
+#[cargo_test]
+fn relative_install_location_with_path_set() {
```

Comment:
Would you mind having this test created before the fix? Otherwise it is a bit hard to tell where there was a bug and we are really fixing it.